apps/bpytop.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# T&M Hansson IT AB © - 2020, https://www.hanssonit.se/
+
+# shellcheck disable=2034,2059
+true
+SCRIPT_NAME="BPYTOP"
```

Comment:
@szaimen Isn't this enough already?

Reply:
Yes it is. Because of this you can remove the SUBTITLE variable now from the whole script.